### PR TITLE
Create spinachleaf's first post with Trinket and markdown formatting

### PR DESCRIPTION
**Pull request by spinachleaf**

### CODE REVIEW

**Comment by spinachleaf** [comment]
Ready for review

File: _posts/spinachleaf/2021-10-20-spinachleaf-trinket-post.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+author: spinachleaf
+title: "spinachleaf's Trinket post"
+---
+Here's the program I'm embedding:
```
Comment (srj1220) [inline _posts/spinachleaf/2021-10-20-spinachleaf-trinket-post.md:6]: I chose the same one!

Comment (spinachleaf) [reply _posts/spinachleaf/2021-10-20-spinachleaf-trinket-post.md:6]: Nice!

**Review by MishaCienfuegos — APPROVED** [review]
Good use of markdown!

**Pull request merged**